### ISSUE TITLE
Incorrect definition of Symbol>>isUnary

### ISSUE THREAD

**Issue by rko281** [dolphinsmalltalk/Dolphin#604]
The latest build of D7.1 adds a definition for Symbol>>isUnary which looks to be incorrect:

isUnary
	"Answer whether the receiver is valid as an infix message selector."

	^self argumentCount == 1

I think this should actually be the definition (albeit a loose one) for isBinary, with isUnary testing for zero arguments.

**Comment by blairmcg** [comment]
True, added in error as part of an incomplete experiment, and should be removed.

**Comment by rko281** [comment]
Thanks for the confirmation; I only noticed because it clashes with a definition in Grease.
